addons/mail/wizard/mail_template_preview.py

```diff
@@ -25,7 +25,7 @@ def default_get(self, fields):
         if not result.get('mail_template_id') or 'resource_ref' not in fields:
             return result
         mail_template = self.env['mail.template'].browse(result['mail_template_id'])
-        res = self.env[mail_template.model_id.model].search([], limit=1)
+        res = self.env[mail_template.sudo().model_id.model].search([], limit=1)
```

Comment:
Can't you use mail_template.model ?

Reply:
not 100% sure why, but with this, I still have the ACL issue, maybe a related field issue :thinking: 
```
mail_template.sudo().model
```

Reply:
It is a related, should be done as sudo ?

Reply:
only if `related_sudo` is set on the field I think

I know now why it failed, because we read `mail_template.model_id.model` just after, and because my fix put the value in cache, it didn't raise after
![screenshot_spect](https://user-images.githubusercontent.com/54316226/175284972-ec4ddfc8-c39a-4d02-bd42-3f17533aa89f.png)